Src/DependencyCollector/NetCore.Tests/DependencyCollectorDiagnosticListenerTests.Netstandard16.cs

```diff
@@ -17,7 +18,7 @@ namespace Microsoft.ApplicationInsights.DependencyCollector
     /// Unit tests for DependencyCollectorDiagnosticListener.
     /// </summary>
     [TestClass]
-    public class DependencyCollectorDiagnosticListenerTests
+    public partial class DependencyCollectorDiagnosticListenerTests
```

Comment:
there are netcoreapp2.0 specific tests in another file, these tests are still applicable